tests/testthat/test-utils_visR.R

```diff
@@ -4,11 +4,11 @@
 
 # Specifications ---------------------------------------------------------------
 
-#' T1. `align_plots()` accepts a list of `ggplot` objects.
+#' T1. `.align_plots()` accepts a list of `ggplot` objects.
```

Comment:
update to reflect the exported function. I think we should remain general eg The function accepts ...